migrations/20160210212844_community_slack.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('community', function(table) {
+    table.text('slack_hook');
+    table.text('slack_team');
+    table.text('slack_configure');
```

Comment:
what do you think about changing this and the `slack_hook` columns to end in `_url` so it's clearer what's to be stored there?